src/connection.ts

```diff
@@ -39,7 +39,10 @@ export const SFDX_HTTP_HEADERS = {
 };
 
 export const DNS_ERROR_NAME = 'Domain Not Found';
-const DNS_TIMEOUT = Math.max(3, new Env().getNumber('SFDX_DNS_TIMEOUT') || 3);
+// Timeout for DNS lookup polling defaults to 30 seconds and should always be at least 3 seconds
+const DNS_TIMEOUT = Math.max(3, new Env().getNumber('SFDX_DNS_TIMEOUT', 30) as number);
+// Retry frequency for DNS lookup polling should be at least 10 seconds
+const DNS_RETRY_FREQ = Math.max(10, new Env().getNumber('SFDX_DNS_RETRY_FREQUENCY', 10) as number);
```

Comment:
Is there a downsize to have the frequency be 3 seconds as opposed to 10 seconds?

Reply:
@amphro The reason it's an issue here is that the timeout and retry frequency were _both_ set to 3, which meant that it only tried the lookup once and then failed.

Now that the timeout defaults to 30, I'm not sure if retrying every 3 seconds would be an issue or not. To be safe I just changed it back to what it used to be in versions 2.16.6 and older